products/BellHybrid/apps/common/src/LanguageUtils.cpp

```diff
@@ -3,21 +3,43 @@
 
 #include <i18n/i18n.hpp>
 
-namespace utils::language
+namespace
 {
-    auto getCorrectMinutesNumeralForm(int val) -> std::string
+    std::string transform(const std::uint32_t val,
+                          const std::string &minute_lower,
```

Comment:
Why use underscore style if camelCase is commonly used?

Reply:
Fixed